### PR TITLE
docs: Update CONTRIBUTING.md to reflect removal of TypeScript

### PR DESCRIPTION
**Pull request by AfolabiOlaoluwa**
This commit updates the `CONTRIBUTING.md` file to reflect the latest changes in the project, specifically the removal of TypeScript. It provides clear instructions and guidelines for contributors on how to contribute to the project now that TypeScript is no longer a part of it.

This makes contributors avoid errors such as using the wrong file extensions to test. 

See reproduced error it fixes in the image below:

Screenshot 2023-09-08 at 20 13 55 1

### CODE REVIEW

**Comment by Ayomide-Daniel** [comment]
@AfolabiOlaoluwa, not sure why you'd delete the `yarn.lock` file. That's why the workflow is failing...

**Comment by AfolabiOlaoluwa** [comment]
Thank you. I'll check.

**Comment by AfolabiOlaoluwa** [comment]
> @AfolabiOlaoluwa, not sure why you'd delete the `yarn.lock` file. That's why the workflow is failing...

Yes, I'll check and append.

File: yarn.lock
Comment (marcoroth) [inline yarn.lock:1]: Is there a reason for deleting the yarn.lock?

Comment (AfolabiOlaoluwa) [reply yarn.lock:1]: It's resolved. Completely oversight. I made the PR from mobile, while on the go.

**Review by Nick-Gabe — APPROVED** [review]

**Pull request merged**